components/organisms/ProfileIdentitySelection.vue

```diff
@@ -29,3 +29,9 @@
 <script lang="ts" setup>
 import RegistrationCard from '~/components/molecules/registration/RegistrationCard.vue'
 </script>
+
+<style lang="scss" scoped>
+.fs-2rem {
+    font-size: 2rem;
+}
```

Comment:
if you find that you have to make more font sizes, why not just extend bootstrap utility like we're already doing in variables.scss

Reply:
Good Idea. Will look into it now